vendor/github.com/minio/minio-go/v6/api-get-object-file.go

```
@@ -0,0 +1,125 @@
+	}
+
+	// Issue Stat to get the current offset.
+	st, err = filePart.Stat()
+	if err != nil {
+		return err
+	}
+
+	// Initialize get object request headers to set the
+	// appropriate range offsets to read from.
+	if st.Size() > 0 {
+		opts.SetRange(st.Size(), 0)
```

golangci-lint (1.64.8)

Error return value of `opts.SetRange` is not checked

(errcheck)